### DISCUSSION TITLE
Guys how can i modify the code that a particular guild id its blacklisted by default

### DISCUSSION THREAD

**Discussion by blaze534new** [JustOptimize/return-ShowHiddenChannels#193] [Q&A] [answered]
like i tried like this 
![image](https://github.com/JustOptimize/return-ShowHiddenChannels/assets/174046419/8a961221-7a07-40f8-b50f-07f12022e452)
but dosent work so if anybody can help i will really appreciate, thanks.

**Comment by JustOptimize** [comment, marked as answer]
What do you want to achive?
If you want to just blacklist a guild you can do it by right-clicking on it and clicking `Disable SHC` or from the plugin settings.
If you want to sync between clients you can copy the config file

I'm kinda against editing the plugin since if an update comes your edits would be overridden.

But if you still belive that editing is the right way for you, you're editing the defaultSettings and it should work but only when generting a NEW settings file.
To override a setting you should foce it after the other settings are loaded like this
```js
this.settings = Utilities.loadData(config.info.name, 'settings', defaultSettings);
this.settings['blacklistedGuilds']['YOUR_ID_HERE'] = true
```
